vmm/src/default_syscalls/aarch64.rs

```diff
@@ -1,22 +1,16 @@
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-use seccomp::{Error, SeccompFilterContext};
+use seccomp::{Error, SeccompAction, SeccompFilter};
 
-/// List of allowed syscalls.
-pub const ALLOWED_SYSCALLS: &[i64] = &[];
-
-/// The default context containing the white listed syscall rules required by `Firecracker` to
+/// The default filter containing the white listed syscall rules required by `Firecracker` to
 /// function.
-pub fn default_context() -> Result<SeccompFilterContext, Error> {
-    Ok(seccomp::SeccompFilterContext::new(
-        vec![].into_iter().collect(),
-        seccomp::SeccompAction::Trap,
-    )
-    .unwrap())
+pub fn default_filter() -> Result<SeccompFilter, Error> {
+    Ok(SeccompFilter::new(vec![].into_iter().collect(), SeccompAction::Trap).unwrap())
 }
 
 /// Applies the configured level of seccomp filtering to the current thread.
+#[allow(unused_variables)]
 pub fn set_seccomp_level(seccomp_level: u32) -> Result<(), Error> {
```

Comment:
You can also do it like this:
```suggestion
pub fn set_seccomp_level(_seccomp_level: u32) -> Result<(), Error> {
```